content/en/docs/contribute/style/style-guide.md

```diff
@@ -359,6 +361,60 @@ Shortcodes inside include statements will break the build. You must insert them
+
+### Headings
+Remember that screen readers are linear output devices which means they output items on a page one at a time, so if there is a lot of content on your page, applying headings gives the page a structure that allows the user to easily navigate, without having to read loads of content that is of no use, value, or interest to them.
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>Use only one first-level heading per page.</td><td>Use first level heading in blog posts, as Hugo will automatically convert the title section of your post’s front matter into a first-level heading.</td></tr><tr><td>Use ordered headings to provide a meaningful high-level outline of your content.</td><td>Use headings level 4 through 6, unless it is absolutely necessary. If your content is that detailed, it may need to be broken into separate articles.</td>
+  <tr><td>Use pound or hash signs (#) for non-blog post content.</td><td> Use underlines (--- or ===) to designate first-level headings.</td></tr> 
+  <tr><td>Use sentence case for headings. For example, <b>Extend kubectl with plugins</b></td><td>Use title case for headings. For example, <b>Extend Kubectl With Plugins</b></td></tr>  
+</table>
+
+### Paragraphs
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>Try to keep the paragraphs on the shorter side. 5 to 6 sentences maximum.</td><td>Indent the first paragraph with space characters. For example, ⋅⋅⋅Three spaces before a paragraph will indent it.</td></tr>
+  <tr><td>Use three hyphens (---) to create a horizontal rule. Use horizontal rules for breaks in paragraph content. For example, a change of scene in a story, or a shift of topic within a section.</td><td>Use horizontal rules for decoration.</td></tr>
+</table>
+
+### Links
+ <table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>Links should provide context for the content they link to. Certain ports are open on your machines. See <a href="#check-required-ports">Check required ports</a> for more details.</td><td>Use ambiguous terms like “click here”. Certain ports are open on your machines. See <a href="#check-required-ports">here</a> for more details.</td></tr> <tr><td>Use Markdown-style links &lpar;&lsqb;link text&rsqb;&lpar;URL&rpar;&rpar;.</td><td>Use HTML style links &lpar;&lt;link href="/media/examples/link-element-example.css" target="_blank"&gt;Visit our tutorial!&rpar; or create links that open in new tabs or windows. For example, &lsqb;example website&rsqb;&lpar;https://example.com&rpar;{target="_blank"}</td></tr>
+</table>
+
+### Lists
+Grouping related items look good visually. When a screen reader comes across a list—whether it is an ordered or unordered list—it will be announced to the user that there is a group of list items. The user can then use the arrow keys to cursor up and down between the various items in the list.
```

Comment:
> Grouping related items look good visually.
I'm afraid I'm not sure what this sentence means.

Reply:
Rephrased the para